pipelines/BII/BII_pipeline.md

```diff
@@ -0,0 +1,74 @@
+# Biodiversity Intactness Index
+### Author(s): Jory Griffith
```

Comment:
Subtitles will not work well on the website. Please use regular text for authors and reviewers.